.gitignore

```diff
@@ -72,5 +72,6 @@ coverage.xml
 # Front-End #
 #############
 node_modules/
+lib/
```

Comment:
Crazy idea: what if we auto-generated an `.npmignore` on prepublish and deleted it on postpublish?